python/pyspark/sql/tests/streaming/test_streaming.py

```diff
@@ -77,8 +109,14 @@ def test_stream_read_options_overwrite(self):
                 .schema(bad_schema)
                 .load(path="python/test_support/sql/streaming", schema=schema, format="text")
             )
-        self.assertTrue(df.isStreaming)
-        self.assertEqual(df.schema.simpleString(), "struct<data:string>")
+            # TODO: Moving this outside of with block will trigger the following error,
```

Comment:
It does not need to be a TODO. Testing it inside `with` is the right thing.
Calls like `schema` are executed on server with the updated state of the conf.

Reply:
I see. So this is a behavior change for migrating to connect?